Content.Server/GameObjects/Components/Items/Storage/ServerStorageComponent.cs

```diff
@@ -288,13 +288,13 @@ public override void HandleNetworkMessage(ComponentMessage message, INetChannel
                         var entity = _entityManager.GetEntity(remove.EntityUid);
                         if (entity != null && storage.Contains(entity))
                         {
-                            Remove(entity);
```

Comment:
It looks a bit now like the item never gets removed from storage.

Reply:
Putting the item in your hand _does_ remove it from storage which is why the Remove() isn't necessary, originally it would always remove the item from storage and place it at your feet when you couldn't put it in your hand which was the behavior I am avoiding here, the only reason Remove() should be included is if it has something to do with sending a message to other players that the storage container doesn't have the item anymore which I can change in a moment

Reply:
What about in the case where there is no `HandsComponent`? Is the item removed from storage then?